go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/golang/mock v1.3.1
 	github.com/golang/protobuf v1.3.2
 	github.com/gorilla/mux v1.7.3
-	github.com/hdac-io/casperlabs-ee-grpc-go-util v0.0.0-20200107075212-e3e40973f879
+	github.com/hdac-io/casperlabs-ee-grpc-go-util v0.0.0-20200129121350-364e7ad6bfca
```

Comment:
How about start version tagging this gRPC util?

Reply:
https://github.com/hdac-io/friday/commit/5a563e48aa5652d229ff068e317be16e11b8ded2